nucleus/core/bootstrap/src/test/java/com/sun/enterprise/glassfish/bootstrap/MainHelperTest.java

```diff
@@ -110,10 +113,13 @@ void createLauncher_Felix() throws Exception {
         Class<?> clazz = loader.loadClass("org.osgi.framework.Bundle");
         assertNotNull(clazz);
         String osgiPackages = cfg.getProperty(FRAMEWORK_SYSTEMPACKAGES);
-        assertThat(osgiPackages,
-            stringContainsInOrder("org.osgi.framework;version=\"1.10\"", "com.sun.jarsigner,", "java.lang,",
-                "java.util.concurrent.locks,", "javax.xml.crypto.dom,", "org.w3c.dom.traversal,"));
-        assertThat(osgiPackages, not(stringContainsInOrder(".hk2.")));
+        assertAll(
+            () -> assertThat(osgiPackages,
+                stringContainsInOrder("org.osgi.framework;version=\"1.10\"", "java.lang,",
+                    "java.util.concurrent.locks,", "javax.xml.crypto.dom,", "org.w3c.dom.traversal,")),
+            () -> assertThat(osgiPackages, not(stringContainsInOrder(".hk2."))), () -> assertThat(osgiPackages,
```

Comment:
It may be a trivial comment, but wouldn't it be clearer to start a new line before the third `() -> assertThat`?

Reply:
You are right, sorry, I did not notice that. Thank you!